docs/ibc-relayer.md

```diff
@@ -0,0 +1,109 @@
+    - Any unbonding requests from that checkpointed epoch are then matured
+    - Given Bitcoin's average block time of ~10 minutes, the average unbonding
+      time is about 50 hours
+
+3. **IBC Light Client Configuration**:
+    - IBC light clients for Babylon Genesis on other chains should have a lower
+      trusting period (~33 hours)
+    - This is about 2/3 of the unbonding period, following standard IBC security
+      practices
+    - This configuration only affects light clients of Babylon Genesis on other
+      chains
+    - The trusting period of other chains' light clients on Babylon Genesis
+      remains unaffected
+
+Due to these unique characteristics, special attention is required when
+configuring the relayer's trusting period and client refresh rate.
+
+## Relayer Configuration
+
+When setting up a relayer for Babylon, pay special attention to these
+parameters:
+
+1. **Trusting Period**: Should be set to approximately 2/3 of Babylon's
+   unbonding period
+    - Babylon's unbonding period is ~50 hours (based on ~300 BTC blocks)
+    - Therefore, the trusting period should be set to ~33 hours
+
+2. **Client Refresh Rate**: A higher refresh rate is recommended (1/5 of
+   trusting period, i.e., ~6.6 hours)
+
+For example, in Hermes configuration:
+
+```
+[[chains]]
+trusting_period = "33 hours"
+refresh = true
+client_refresh_rate = 1/5
+```
+
+For complete setup instructions, including wallet configuration, connection
+setup, and channel creation, refer to:
+
+- [Celestia's IBC Relayer Guide](https://docs.celestia.org/how-to-guides/ibc-relayer)
+- [Osmosis's Relayer Guide](https://docs.osmosis.zone/osmosis-core/relaying/relayer-guide)
+
+## Monitoring and Maintenance
+
+Regular monitoring of your IBC clients is crucial. For example, if using Hermes,
+you can monitor the `client_updates_submitted_total` metric, which counts the
+number of client update messages submitted between chains. This metric should
+increase over time as your relayer submits updates to keep the IBC clients
+synchronized. For detailed information about this metric as well as other
+important metrics, refer
+to [Hermes metrics documentation](https://hermes.informal.systems/documentation/telemetry/operators.html#what-is-the-overall-ibc-status-of-each-network).
+
+> **Note**: For advanced monitoring,
+> see [Informal Systems' IBC Insights](https://insights.informal.systems/noble/osmosis).
+
+## Handling Expired/Frozen IBC Clients
+
+If an IBC client expires or becomes frozen, you'll need to submit a governance
+proposal to recover the client. This proposal needs to be submitted on the chain
+that maintains the light client of the counterparty chain.
+
+For example, if you're relaying between Babylon and another chain:
+
+- If Babylon's light client of the other chain expires, submit the proposal on
+  Babylon
+- If the other chain's light client of Babylon expires, submit the proposal on
+  the other chain
```

Comment:
This seems not right -- I thought

```suggestion
- If Babylon's light client of the other chain expires, submit the proposal on
  that chain
- If the other chain's light client of Babylon expires, submit the proposal on
  Babylon
```

Can we double check?

Reply:
Hmm I see your point. "Babylon's light client of the other chain" means The other chain's IBC light client inside Babylon chain. Shall we refactor the sentence to make it more clear?